3d_segmentation/swin_unetr_btcv_segmentation_3d.ipynb

```diff
@@ -258,188 +288,18 @@
-    "        },\n",
-    "        {\n",
-    "            \"image\": \"imagesTr/img0031.nii.gz\",\n",
-    "            \"label\": \"labelsTr/label0031.nii.gz\"\n",
-    "        },\n",
-    "        {\n",
-    "            \"image\": \"imagesTr/img0032.nii.gz\",\n",
-    "            \"label\": \"labelsTr/label0032.nii.gz\"\n",
-    "        },\n",
-    "        {\n",
-    "            \"image\": \"imagesTr/img0033.nii.gz\",\n",
-    "            \"label\": \"labelsTr/label0033.nii.gz\"\n",
-    "        },\n",
-    "        {\n",
-    "            \"image\": \"imagesTr/img0034.nii.gz\",\n",
-    "            \"label\": \"labelsTr/label0034.nii.gz\"\n",
-    "        }\n",
-    "    ],\n",
-    "    \"validation\": [\n",
-    "        {\n",
-    "            \"image\": \"imagesTr/img0035.nii.gz\",\n",
-    "            \"label\": \"labelsTr/label0035.nii.gz\"\n",
-    "        },\n",
-    "        {\n",
-    "            \"image\": \"imagesTr/img0036.nii.gz\",\n",
-    "            \"label\": \"labelsTr/label0036.nii.gz\"\n",
-    "        },\n",
-    "        {\n",
-    "            \"image\": \"imagesTr/img0037.nii.gz\",\n",
-    "            \"label\": \"labelsTr/label0037.nii.gz\"\n",
-    "        },\n",
-    "        {\n",
-    "            \"image\": \"imagesTr/img0038.nii.gz\",\n",
-    "            \"label\": \"labelsTr/label0038.nii.gz\"\n",
-    "        },\n",
-    "        {\n",
-    "            \"image\": \"imagesTr/img0039.nii.gz\",\n",
-    "            \"label\": \"labelsTr/label0039.nii.gz\"\n",
-    "        },\n",
-    "        {\n",
-    "            \"image\": \"imagesTr/img0040.nii.gz\",\n",
-    "            \"label\": \"labelsTr/label0040.nii.gz\"\n",
-    "        }\n",
-    "    ]\n",
-    "}\n",
-    "    "
+    "## Download dataset and format in the folder\n",
+    "1. Download dataset from here: https://www.synapse.org/#!Synapse:syn3193805/wiki/89480. After you open the link, navigate to the \"Files\" tab, then download Abdomen/RawData.zip.\n",
+    "\n",
+    "    Note that you may need to register for an account on Synapse and consent to use agreements before being able to view/download this file. There are options to download directly from the browser or from the command line; please refer to Synapse API documentation for more info.\n",
+    "\n",
+    "\n",
+    "2. After downloading the zip file, unzip. Then put images from `RawData/Training/img` in `./data/imagesTr`, and put labels from `RawData/Training/label` in `./data/labelsTr`.\n",
+    "\n",
+    "\n",
+    "3. Make a JSON file to define train/val split and other relevant parameters. Place the JSON file at `./data/dataset_0.json`.\n",
+    "\n",
+    "    You can download an example of the JSON file [here](https://drive.google.com/file/d/1t4fIQQkONv7ArTSZe4Nucwkk1KfdUDvW/view?usp=sharing), or, equivalently, use the following `wget` command. If you would like to use this directly, please move it into the `./data` folder."
```

Comment:
May I know where this google drive file is? Is it in the MONAI google drive storage?

Thanks.

Reply:
I added this file to the MONAI google drive and updated the links. Thanks!